tests/node/rpc_test.go

```diff
@@ -127,10 +126,8 @@ func p2pBanPeer(t devtest.T, out *MixedOpKonaPreset) {
 		// We pick the first peer that is connected.
 		peerToBan := ""
 		for _, peer := range peers.Peers {
-			if peer.Connectedness == network.Connected {
-				peerToBan = peer.PeerID.String()
-				break
-			}
+			peerToBan = peer.PeerID.String()
+			break
 		}
 
```

Comment:
Why even loop here? just grab the first peer id string and just require that the list is not empty